src/Hl7.Fhir.Specification/Specification/Snapshot/ElementDefnMerger.cs

```diff
@@ -340,6 +340,27 @@ private FhirString deepCopyAndRaiseOnConstraint(FhirString elt)
                     return snap;
             }
 
+            private ElementDefinition.ElementDefinitionBindingComponent mergeBinding(ElementDefinition.ElementDefinitionBindingComponent snap, ElementDefinition.ElementDefinitionBindingComponent diff)
```

Comment:
Add some comments why we introduce here a new method for Binding. For feature reference.